jwst/assign_wcs/tests/data/step_SourceCatalogStep_cat.ecsv

```diff
@@ -23,6 +23,7 @@
 # - {name: orientation_sky, unit: deg, datatype: float64}
 # - {name: isophotal_abmag, datatype: float64}
 # - {name: isophotal_abmag_err, datatype: float32}
+# - {name: is_star, datatype: float64}
```

Comment:
Perhaps you should modify the data file used in the test to have the `is_star` column be Boolean and modify the entries in the ecsv file itself to also be boolean for that column (perhaps with a mixture of True and False values).